crates/backend/src/ast.rs

```diff
@@ -230,6 +249,7 @@ impl Export {
         Ident::new(&generated_name, Span::call_site())
     }
 
+    /// ???
```

Comment:
This is the name of the shim function that gets exported and takes the raw ABI form of its arguments and converts them back into their normal, "high level" form before calling the actual function.